### DISCUSSION TITLE
Nuxt 3 Access data offline from Firebase

### DISCUSSION THREAD

**Discussion by OptoCode** [vuejs/vuefire#1387] [Questions and Help] [answered]
I am trying to enable Firebase offline access in my Nuxt 3 project. I thought maybe there is an option I could enable in the nuxt.config.ts. I was going to create a plugin. If I could get some advice on a better way to do this I would appreciate it. Thank you in advance!

Here is what I was thinking:
vuefire: {
    persistance: true,
    auth: true,
    config: {....}
}

**Comment by posva** [comment]
What is the persistance you want to activate? Firebase should support offline by default: https://firebase.google.com/docs/database/web/offline-capabilities

**Reply by posva** [reply, marked as answer]
oh you mean this: https://github.com/vuejs/vuefire/issues/1351 ?

**Reply by OptoCode** [reply]
Thanks, I am going to try that. Appreciate!!

**Comment by Triloworld** [comment]
@posva We don't have it running with auth as you mention. Where we can enable it?
Issue is that documentation mention enabling on instance and module isn't doing it - https://firebase.google.com/docs/database/web/offline-capabilities
We try to use but not working ;/
